client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -526,6 +541,12 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         SITE_LOCALE("siteLocale", "US", false),
 
         /**
+         * Site will show country used for tests.<br>
+         * Default is set to <b>UNDEFINED</b>
+         */
+        MOBILE_PLATFORM("mobilePlatform", "UNDEFINED", false),
```

Comment:
I don't see anywhere in this PR where this new `ConfigProperty` is used. Am I missing something?